internal/controllers/vgmanager/controller_test.go

```diff
@@ -844,11 +858,10 @@ func testReconcileFailure(ctx context.Context) {
 	})
 
 	By("triggering wipefs failure", func() {
-		evalSymlinks = func(path string) (string, error) {
-			return path, nil
-		}
+		instances.Reconciler.SymlinkResolveFn = func(path string) (string, error) { return path, nil }
+
 		defer func() {
-			evalSymlinks = filepath.EvalSymlinks
+			instances.Reconciler.SymlinkResolveFn = filepath.EvalSymlinks
```

Comment:
now that we no longer set a global variable, can we set it in the instances.Reconciler creation? that way we dont have to defer reset and we dont need to set them up everytime. WDYT?

Reply:
All tests should work with that symlink resolution so you can do it once per suite. There is no need to revert it at all I believe